governance/reviews/template.md

```diff
@@ -23,171 +11,207 @@ Exemplary: project has an extraordinary level of governance development and impl
+does it have most of the project roles that actually exist?  Do those roles form a reasonable set
+of steps from new contributor to maintainer?  Does each role have qualifications, duties, and powers 
+defined?
+If the project does not have one, suggest that TAG-CS could help them create one. --->
 
-Recent changes to the governance include:
+* Clearly defined and discoverable process to submit issues or changes.
 
-*
+<!-- Files: contributing.md, reviewing.md, release.md, other files which form a
+howto for contributors.  Check for the presence of these and their completeness;
+as a project grows it should be adding more and more detailed content to its contributor
+docs. 
+If you have time or reason to, spot check issue and PR activity on the project's main repos,
+and the devstats charts related to contributions.  Are contributions by non-maintainers
+handled or ignored?  Is the project accumulating a backlog of unreviewed contributions, or
+concluding them? --> 
 
-Areas of potential future development include:
+* Project must have, and document, at least one public communications channel for users and/or contributors.
+* List and document all project communication channels, including subprojects (mail list/slack/etc.).  List any non-public communications channels and what their special purpose is.
 
-*
+<!-- audit the list of communications channels supplied.  Does it include at least one
+general channel?  Does it include all channels you know of?  Are those channels public/
+owned by the CNCF, or are some of them private channels controlled by someone else? -->
 
-### Governance Findings Table
-<!--- Add additional rows as necessary. For each finding described above, it should also be included here with further detail. 
+* Up-to-date public meeting schedulers and/or integration with CNCF calendar.
 
-Should be reported in the "Critical Items" section:
-- Critical
+<!-- Does the project have a calendar (ideally integrated with the CNCF calendar)
+that lists all documented meetings?  Are there archival videos and minutes kept
+of these meetings?  Are the meeting agendas findable and useful? --> 
 
-Should be reported as "Areas for Improvement"
-- Medium/low: needs improvement
+* Documentation of how to contribute, with increasing detail as the project matures.
 
---->
+<!-- Files: contributing.md, reviewing.md, release.md, other files which form a
+howto for contributors.  Check for the presence of these and their completeness;
+as a project grows it should be adding more and more detailed content to its contributor
+docs. Cite missing parts based on the level of maturity of the project. --->
+
+* Demonstrate contributor activity and recruitment.
+
+<!-- Check provided evidence Spot check issue and PR activity on the project's main repos,
+and the devstats charts related to contributions.  Are contributions by non-maintainers
+handled or ignored?  Is the project accumulating a backlog of unreviewed contributions, or
+concluding them? --> 
```

Comment:
i think the portion of this we dont have good coverage on is the "Recruitment" aspect.  for me is look to see if they're inviting people to meetings, inviting to participate, presence at KCCN, etc. thoughts on including this check or for now lets hold off?

Reply:
Well, recruitment is easy to assess if it's successful; you can see it through steadily increasing numbers of new contributors.  It's more qualitative if it's not very successful, but then feedback is required anyways.

Reply:
gotcha. thanks for the clarification!